packages/@lwc/engine-core/src/3rdparty/snabbdom/types.ts

```diff
@@ -15,7 +15,7 @@
 
 import { VM } from '../../framework/vm';
 
-export type VNodeStyle = Record<string, string>;
+export type VNodeStyleDecls = Array<[string, string] | [string, string, boolean]>;
```

Comment:
I suggest we set the third value to `false` instead of leaving it `undefined`. This way we can just pass the value as-is when invoking`renderer.setCSSStyleProperty()` later on.
```suggestion
export type VNodeStyleDecls = Array<[string, string, boolean]>;
```

Reply:
The reason I avoided this is because I figured most style declarations would not have `!important`, so I didn't want to generate a bunch of `, false`s in the template. Maybe this isn't a big deal though; minify will turn it into `!1` and gzip will dedup it.

Reply:
OK done, it's always an array of length 3 now.